crawl-ref/source/dat/descript/spells.txt

```diff
@@ -345,6 +345,12 @@ Corrosive Bolt spell
 
 Fires a penetrating bolt of acid.
 %%%%
+Corrupt Locale spell
+
+Tears apart the reality of the dungeon in the immediate vicinity, allowing
+the denziens and horrors of the Abyss to permanently seep through. Followers
+of Lugonu will find such abominations neutral to their cause.
```

Comment:
Is this a fun mechanic? As with Vaud, having gods give you protection against specific monsters does not seem like a particularly good idea to me, and it's not like Lugonu gives you protection against abyss monsters in general. Are you sure this is a good idea?

Reply:
Yeah fair challenge on that, theme may well > fun on that one. Angry monsters for everyone!